app/models/board.rb

```diff
@@ -46,4 +48,12 @@ def set_slug
   def send_action_items
     DailyActionItemsJob.set(wait: 1.day).perform_later(self)
   end
+
+  def default_retro_date
+    self.retro_date ||= created_at
```

Comment:
Why do you need conditional assignment operator, if this method will be called only if retro date is nil